runtime/cudaq/platform/mqpu/nvidia-mqpu.config

```diff
@@ -8,7 +8,7 @@
 
 msg=""
 gpu_found=$(query_gpu)
-if ${gpu_found} && [ -f "${install_dir}/lib/libnvqir-custatevec-fp32.so" ]; then
+if ${NVQC} || (${gpu_found} && [ -f "${install_dir}/lib/libnvqir-custatevec-fp32.so" ]); then
```

Comment:
Does this mean that `--nvqc --nvcq-backend=nvidia-mqpu` is now a valid combination of arguments?

Reply:
> Does this mean that `--nvqc --nvcq-backend=nvidia-mqpu` is now a valid combination of arguments?

The `--nvqc-backend` argument has been removed in this PR per team suggestions, but if you want to use the `nvidia-mqpu` target, you would now specify it as `--nvqc --target nvidia-mqpu`. Additionally, you would need to specify the number of GPUs, so it would really need to be `--nvqc --nvqc-ngpus <N> --target nvidia-mqpu`